### DISCUSSION TITLE
ToS Discord

### DISCUSSION THREAD

**Discussion by PM-KIRILL** [Najmul190/Discord-AI-Selfbot#65] [Ideas]
Please add a filter for certain phrases, as the bot might be prompted to say things that violate Discord's Terms of Service. For example, someone could make the bot say: "I am 10 years old." Discord could block the bot for this phrase.

**Comment by Najmul190** [comment]
Added, you can now turn on the option to just remove any numbers below 13

**Reply by PM-KIRILL** [reply]
Thanks)
